### PR TITLE
Fixing node selector and toleration issue while generating the delegate manifest

### PR DESCRIPTION
**Pull request by imrajdas**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by Jonsy13 — APPROVED** [review]

File: litmus-portal/graphql-server/manifests/namespace/3a_agents_rbac.yaml [hunk 1]
```diff
@@ -48,11 +48,11 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
   name: subscriber-role-binding
-  namespace: #{AGENT-NAMESPACE}
+  namespace: #{AGENT_NAMESPACE}
 subjects:
   - kind: ServiceAccount
     name: #{SUBSCRIBER-SERVICE-ACCOUNT}
```
Comment (amityt) [inline litmus-portal/graphql-server/manifests/namespace/3a_agents_rbac.yaml:54]: `SUBSCRIBER-SERVICE-ACCOUNT`

File: litmus-portal/graphql-server/manifests/namespace/3b_agents_deployment.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: agent-config
-  namespace: #{AGENT-NAMESPACE}
+  namespace: #{AGENT_NAMESPACE}
 data:
   SERVER_ADDR: #{SERVER_ADDR}
   AGENT_SCOPE: #{AGENT-SCOPE}
```
Comment (amityt) [inline litmus-portal/graphql-server/manifests/namespace/3b_agents_deployment.yaml:9]: `AGENT_SCOPE`

File: litmus-portal/graphql-server/manifests/namespace/3b_agents_deployment.yaml [hunk 3]
```diff
@@ -40,12 +40,12 @@ spec:
       labels:
         app: subscriber
     spec:
-      #{tolerations}
-      #{nodeselector}
+      #{TOLERATIONS}
+      #{NODE_SELECTOR}
       serviceAccountName: #{SUBSCRIBER-SERVICE-ACCOUNT}
```
Comment (amityt) [inline litmus-portal/graphql-server/manifests/namespace/3b_agents_deployment.yaml:45]: `SUBSCRIBER_SERVICE_ACCOUNT`

**Review by amityt — COMMENTED** [review]
Minor changes required

**Review by SarthakJain26 — APPROVED** [review]

**Review by amityt — APPROVED** [review]

**Pull request merged**